### ISSUE TITLE
install problem

### ISSUE THREAD

**Issue by jiangshan529** [ay-lab/fithic#63]
Hi, Dear all,

I am trying to install fithic with conda(conda install -c "bioconda/label/cf201901" fithic). However, it always report errors like this:

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: / 
Found conflicts! Looking for incompatible packages.                                                                                                                       failed                                                                                                                                                                        

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - fithic -> python[version='>=3']

Your python: python=2

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.

The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.19=0
  - feature:|@/linux-64::__glibc==2.19=0

Your installed version is: 2.19

Please let me know how to deal with this.
Thanks!

**Comment by ay-lab** [comment]
Hi @jiangshan529 
Thanks for your query. It seems you have python2 installed in your conda environment. Please create a conda environment with python3 (like python 3.7) and then install fithic. Let me know if you face any issues.

**Comment by jiangshan529** [comment]
Hi, I create a new conda environment with python3.7(conda create -n fithic python=3.7). However, the installation still shows such error.

> Hi @jiangshan529 Thanks for your query. It seems you have python2 installed in your conda environment. Please create a conda environment with python3 (like python 3.7) and then install fithic. Let me know if you face any issues.

**Comment by aryakaul** [comment]
Can you confirm that you've activated the new environment and python v3 is active? (paste the output of ``python --version``)

**Comment by jiangshan529** [comment]
Yes, I activate the envrionment with "source activate fithic". When I print python, it shows "Python 3.7.13".

> Can you confirm that you've activated the new environment and python v3 is active? (paste the output of `python --version`)

**Comment by aryakaul** [comment]
can you post a screenshot with the error you get when you run `conda install fithic`

**Comment by jiangshan529** [comment]
Hi, hope you can see the picture through email:

[image: image.png]
[image: image.png]
[image: image.png]

Arya ***@***.***> 于2022年10月15日周六 13:04写道：
